src/components/Directory/List.js

```diff
@@ -1435,13 +1400,6 @@ export const list = [
     description:
       "Roverba offers cloud hosting and telecom services to businesses. Address: 7 rue des Maraichers, 69120 Vaulx-en-Velin."
   },
-  {
```

Comment:
This one seems to be still active?

Reply:
I guess I thought its offline because the site doesn't support TLS. Also

Reply:
Should we bring them back and remove the rest, wdyt?

Reply:
brought back, done